heartbeat/ocf-shellfuncs.in

```diff
@@ -426,7 +426,7 @@ ocf_run() {
 		"-q")
 		    verbose=""
 		    shift 1;;
-		"-info"|"-warn"|"-err")
+		"-info"|"-warn"|"-err"|"-debug")
 		    loglevel=`echo $1 | sed -e s/-//g`
```

Comment:
```suggestion
		    loglevel=${loglevel#-}
```
Would avoid a spurious sub process.
But we don't have to fix all of that nonsense "it's still 1981" shell coding style now ;-)
Though someone should eventually.